### ISSUE TITLE
command is named slightly differently on Windows

### ISSUE THREAD

**Issue by borkdude** [taylorwood/clj.native-image#8]
On Windows I need to pass in a different name for `native-image`:

```
C:\\Users\\IEUser\\Downloads\\graalvm\\graalvm-ce-19.0.0\\bin\\native-image.cmd
```

It would be more ergonomic if clj.native-image could automatically detect we're on Windows and derive the command name from the `GRAALVM_HOME` variable. 

Detecting the OS name can be done as follows:

```
(System/getProperty "os.name") ;;=> "Windows 10"
```

**Comment by borkdude** [comment]
This is fixed with https://github.com/taylorwood/clj.native-image/pull/9

**Comment by taylorwood** [comment]
Fixed in 546a77a4492be2f1d9b48e43cb48fb0ebfa39cc2.
